docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```diff
@@ -1960,13 +2316,50 @@ code: |
   
   remove_tmp_files = True
 ---
+need:
+  - root_path
+  - alkiln_session_id
+  - format_error
+id: remove alkip session folder
+code: |
+  def delete_kip_session_dir():
+    """
+    Try to remove ALKiP runtime session folder.
+    Returns:
+        (dict):
+          ok (bool): False if there was an error. Else True.
+          message (str): Any message about the operation.
+    """
+    import shutil
+  
+    try:
+      # Remove ALKiP runtime vars file
+      shutil.rmtree(f"{ root_path }/{ alkiln_session_id }")
+  
+    except Exception as delete_session_dir_error:
+      message1 = f'🖊️ ALKiln NOTE: skipped removing "{ root_path }/{ alkiln_session_id }".'
+      message2 = format_error( delete_session_dir_error )
+      _log( message1 )
+      _debug( message2 )
+      return { "ok": False, "message": f"{ message1 }\n{ message2 }" }
+  
+    return { "ok": False, "message": "" } 
+---
+code: |
+  def format_error( error ):
+    return "\n".join(traceback.format_exception( error ))
```

Comment:
I'm not sure this is worth it, especially since I think this traceback may not work the way I want when it's here. I'll have to delete my use of it elsewhere.